crates/api_models/src/payments.rs

```diff
@@ -5273,6 +5326,8 @@ pub struct OrderDetailsWithAmount {
     pub quantity: u16,
     /// the amount per quantity of product
     pub amount: MinorUnit,
+    pub tax_rate: Option<i64>,
```

Comment:
why do we need to add these 2 fields?

Reply:
when you say this is a `rate` it is expected to be a floating type right? I can provide the rate as 1.2 or 6.8. If there is any specific reason as to why this is restricted to i64, please provide clear documentation

Reply:
When I am making tax_rate to be Option<f64>, Eq will have to be removed from #derive or  trait `std::cmp::Eq` have to be implemented manually. Will that be good to remove Eq from #derive and make tax_rate to be <f64>?